configs/src/networks.json

```diff
@@ -0,0 +1,84 @@
+{
+  "mainnet": {
+    "url": "https://mainnet.infura.io/v3/481a4cdc7c774286b8627f21c6827f48",
+    "isTestnet": false,
+    "chainId": 1,
+    "aliases": {
+      "ethers5": "homestead"
```

Comment:
Should we add an alias for satusuma/alchemy, because networks name may change there also

Reply:
we try to stick to the same naming in the subgraph names
![image](https://github.com/aragon/osx-commons/assets/4057473/90ad5327-81e7-43c5-849d-752ab250e579)

What names do you mean? the ones part of the manifest?

Reply:
this ones https://docs.satsuma.xyz/hosted-subgraphs/supported-chains

Reply:
yeah these are part of the manifests and don't belong to the commons from my point of view because we cannot assume other devs are using alchemy subgraphs

Reply:
But we also want to kill all the mappings that the SDK had to make.

I am with Jose here to put these aliases in one place and this seems to be the right place for me.

Reply:
we can have them this way and if we see that the SDK has to do mapping we'll add them in the future

Reply:
don't see a point because the subgraph doesn't expose the internal chain names and the SDK shouldn't need those names, plus it adds back to duplication because these names are used in the manifest and make generation of the manifest a little harder.

Added it because 2 against 1